crates/wasm/src/lib.rs

```diff
@@ -54,6 +54,9 @@ pub fn compile(args: JsValue) -> JsValue {
 
     driver.create_local_crate(path, CrateType::Binary);
 
+    // We are always adding std lib implicitly. It comes bundled with binary.
+    add_noir_lib(&mut driver, (&"std").to_string());
```

Comment:
```suggestion
    add_noir_lib(&mut driver, "std");
```
We can change `add_noir_lib` to take a `&str` which should make things clearer.